bitrise.yml

```diff
@@ -1542,8 +1542,9 @@ workflows:
             $HOME/google-cloud-sdk/bin/gcloud version
             $HOME/google-cloud-sdk/bin/gcloud auth activate-service-account -q --key-file /tmp/key-file.json
             $HOME/google-cloud-sdk/bin/gcloud config set project "$GOOGLE_CLOUD_PROJECT"
-            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run --test /Users/vagrant/git/Fennec_Enterprise.zip \
-                --model=iphone14pro,version=16.4 \
+            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run \
+                --test /Users/vagrant/git/Fennec_Enterprise.zip \
+                --device model=iphone14pro,version=16.4 \
                 --xcode-version=14.3.1
```

Comment:
I haven't seen xcode 14.3.1 on Firebase, just 14.3

Reply:
it worked during my testing, but i won't risk it. mightve just been my sandbox. updating now.

Reply:
actually, i'm removing it since it's optional. don't know why i included; latest should always be used by default.